BrainPortal/app/controllers/neurohub_application_controller.rb

```diff
@@ -83,6 +84,36 @@ def check_if_locked
 
   end
 
+  ########################################################################
+  # Messaging System Filters (presently only invite acceptance)
+  ########################################################################
+
+  # Find new messages to be displayed at the top of the page.
+  def prepare_messages #:nodoc:
+    return unless current_user
+    return if     current_user.all_licenses_signed.blank?
+    return if     request.format.blank? || request.xhr?
+    return unless request.format.to_sym == :html || params[:controller] == 'messages'
```

Comment:
well... I think putting && it will break the functionality (mind `unless`). As a compromise I can remove or comment out  the second part of the condition since, at the moment, there is no messages controller in NeuroHub part at the moment.